util/util.go

```diff
@@ -643,6 +643,33 @@ func CreateDiskPathReplicaSubdirectory(path string) error {
 	return nil
 }
 
+func DeleteDiskPathReplicaSubdirectoryAndDiskCfgFile(path string) error {
```

Comment:
This part of code change in util.go addresses a different issue, move it to a separate commit.